geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -168,11 +168,8 @@ object RichAttributeDescriptors extends Conversions {
 
     private def tryClass(value: AnyRef): Class[_] = Try(Class.forName(value.asInstanceOf[String])).getOrElse(null)
 
-    def isList: Boolean = ad.getUserData.containsKey(UserDataListType)
```

Comment:
Is this change going to mess with any AttributeDescriptor data that's serialized somewhere?

Reply:
it should be functionally equivalent... it should be impossible to have a list-type attribute without the user-data-list-type key, and vice-versa. it just seemed more logical to check based on the attribute type.

Reply:
Cool!  Just making sure we weren't changing the 'API' in some sense.